test/unit/openshift/component.test.ts

```diff
@@ -106,10 +111,11 @@ suite('OpenShift/Component', () => {
             sandbox.stub(Progress, 'execCmdWithProgress').resolves();
             progressFunctionStub = sandbox.stub(Progress, 'execFunctionWithProgress').yields();
             sandbox.stub(vscode.workspace, 'workspaceFolders').value([wsFolder1, wsFolder2]);
+
         });
 
         test('returns null when cancelled', async () => {
-            quickPickStub.onFirstCall().resolves();
+            quickPickStub.onFirstCall().resolves(null);
```

Comment:
`window.showQuickPick` AFAIK newer returns null it ether instance of string or undefined.